### PR TITLE
feat: add notebook changes to the notification bar

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

I noticed that title wasn't in the notebook activity log... and so realised that we don't share the activity log for them anywhere

## Changes

Adds notebooks to the notification bell if someone edits a notebook you created

## How did you test this code?

developer tests and 👀 locally

Screenshot 2023-07-03 at 18 02 15

### CODE REVIEW

File: frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    },
+}
+
+function nameAndLink(logItem?: ActivityLogItem): JSX.Element {
+    return logItem?.detail?.short_id ? (
+        <Link to={urls.notebook(logItem.detail.short_id)}>{logItem?.detail.name || 'unknown'}</Link>
+    ) : logItem?.detail.name ? (
+        <>{logItem?.detail.name}</>
+    ) : (
+        <>unknown</>
```
Comment (daibhin) [inline frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx:31]: Will the name be unknown if the title does not exist? Perhaps we should call it `Untitled` as we do for new notebooks

File: frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    )
+}
+
+export function notebookActivityDescriber(logItem: ActivityLogItem): HumanizedChange {
+    if (logItem.scope !== ActivityScope.NOTEBOOK) {
+        console.error('notebook describer received a non-Notebook activity')
```
Comment (daibhin) [inline frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx:37]: Should we capture this error anywhere?

**Review by daibhin — APPROVED** [review]

File: frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx (hunk 1 above)
Comment (pauldambra) [reply frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx:31]: That's way better 🙌

File: frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx (hunk 2 above)
Comment (pauldambra) [reply frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx:37]: It's slightly laziness... it's there to help the future traveller adding a new kind of describer. _Really_ the problem should be refactored away

File: frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx (hunk 1 above)
Comment (pauldambra) [inline frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx:31]:
```suggestion
        <i>Untitled</i>
```

**Pull request merged**